### PR TITLE
Workflow: describing old channels removal

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Converting the contribution from the uyuni board related to best practices for removing old channels.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/2363
- [ ] Manager-4.2

# Links

Fixes https://github.com/SUSE/spacewalk/issues/19977 / https://github.com/uyuni-project/uyuni-docs/issues/1940.

### CODE REVIEW

File: modules/common-workflows/pages/workflow-removing-channel.adoc [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
++
+----
+spacewalk-remove-channel -a sles12-sp5-pool-x86_64
+----
+
+
+[[cleaning-up-channel-packages]]
+=== Cleaning up channel packages
+
+==== Running the sync
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-removing-channel.adoc:111]: Instead of "sync" better use synchronization.

**Review by keichwa — COMMENTED** [review]
I'll continue later...

**Comment by 0rnela** [comment]
@keichwa -- thanks, please see the corrections.

File: modules/common-workflows/pages/workflow-removing-channel.adoc (hunk 1 above)
Comment (0rnela) [reply modules/common-workflows/pages/workflow-removing-channel.adoc]: done, thanks

File: modules/common-workflows/pages/workflow-removing-channel.adoc [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+* Be careful not to remove channels that systems are currently subscribed to, or that you are planning on subscribing a system to.
+* If there are systems that are currently subscribed to a channel that you want to delete, do not remove the channel until those systems are upgraded or unsubscribed.
+====
+
+
+==== Identifying Channel labels
+
+.Procedure: Identifying channel labels using {webui}
+[role=procedure]
+. In the SUSE Manager {webui}, in the sidebar navigate to menu:Software[Channel List > All]
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-removing-channel.adoc:37]:
```suggestion
. In the {productname} {webui}, navigate to menu:Software[Channel List > All].
```
The next "steps" are not always steps, and the procedure sounds unnecessarily complex.  I'd vote to do the thing first completely with the webui and then the complete thing on the command line.

File: modules/common-workflows/pages/workflow-removing-channel.adoc [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+
+[[cleaning-up-channel-packages]]
+=== Cleaning up channel packages
+
+==== Running the synchronization
+
+.Procedure: Running the synchronization
+. After removing a channel, run the command ``spacewalk-repo-sync`` on all remaining channels, or wait for the operation to complete automatically.
+. Alternatively, for all currently added channels running the following command:
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-removing-channel.adoc:116]: This workflow isn't about adding channels.  Thus I wouldn't mention it here.

**Comment by keichwa** [comment]
I think it would  be good if @slsnow who initially proposed this enhancement could help reviewing it.

**Review by keichwa — APPROVED** [review]
LGTM, thanks Ornela!

**Pull request merged**